standalone/gradle.properties

```
@@ -3,10 +3,10 @@ artifactory_contextUrl=https://artifactory.labkey.com/artifactory
 
 # Version 1.17.0 or higher is required to fully support building stand-alone modules with the LabKey Gradle plugins
 # Exact version will vary based on the version of LabKey being built on
-gradlePluginsVersion=1.27.0_standaloneBuild-SNAPSHOT
+gradlePluginsVersion=1.27.0
 
 # versions of artifacts required as dependencies
-labkeyVersion=21.3.4
+# labkeyVersion=21.3.4 #disabled to work with standard build
```

```suggestion
#uncomment the version number and set to the desired labkeyVersion
#labkeyVersion=21.3.4
```
Adding the comment at the end of the line and the space at the beginning makes it more difficult to uncomment and get to the right thing.

There should probably be an update to docs or an error message so folks know what they need to do with this to make it work as a standalone module.

---

I added an helpful message to settings.gradle